docs/computer/student_Reports/yash_pranshi_saloni_2020.md

```diff
@@ -0,0 +1,90 @@
+
+| Title   |      Author     |  Email-Id  |
+|:----------:|:-------------:|:------:|
+| Report for the tech stack learned till now at AUV club, ZHCET | Yash Agarwal | yashagarwal1214@gmail.com |
```

Comment:
Do not make a table. THe format is to be in YAML.
---
title:
author:
email-id
---